src/main/kotlin/apply/ui/admin/mission/MissionPreviewDialog.kt

```diff
@@ -0,0 +1,55 @@
+package apply.ui.admin.mission
+
+import com.vaadin.flow.component.Component
+import com.vaadin.flow.component.Html
+import com.vaadin.flow.component.button.Button
+import com.vaadin.flow.component.dialog.Dialog
+import com.vaadin.flow.component.html.H2
+import com.vaadin.flow.component.orderedlayout.FlexComponent
+import com.vaadin.flow.component.orderedlayout.HorizontalLayout
+import com.vaadin.flow.component.orderedlayout.VerticalLayout
+import support.views.createContrastButton
+
+class MissionPreviewDialog(
+    htmlString: String
+) : Dialog() {
+    init {
+        add(createHeader(), createHtmlComponentFrom(htmlString), createButtons())
+        width = "700px"
+        height = "800px"
+        open()
+    }
+
+    private fun createHeader(): VerticalLayout {
+        return VerticalLayout(H2("과제 설명 미리보기")).apply {
```

Comment:
c: "과제 미리 보기" 또는 메일 미리 보기와 합치면 "미리 보기"도 좋습니다.

```suggestion
        return VerticalLayout(H2("미리 보기")).apply {
```